ruby/vampires.rb

```diff
@@ -0,0 +1,84 @@
+#How many people are you going to be hired?
+puts "How many people are going to be hired?"
+hired = gets.chomp.to_i
+
+#Begin hired loop
+until hired == 0
+  # what's your name?
+  puts "What's your name?"
+  name = gets.chomp
+
+  #How old are you? What year were you born?
+  puts "How old are you?"
+  age = gets.chomp.to_i
+  puts "What year were you born?"
+  birth_year = gets.chomp.to_i
+  #Checking age in 2016
+  age_check = 2016 - birth_year - age
+  if age_check > 1
+    age_check = false
+    result = -1
+  elsif age_check < 0
+    age_check = false
+    result = -1
+  else
+      age_check = true
+      result = 1
+  end
+    #debugging
+  puts age_check
+  puts result
+
+  #Should we order you garlic bread?
+  puts "Would you like an order of garlic bread?"
+  garlic = gets.chomp
+  case garlic
```

Comment:
I think you would be better served with `if/else` here case statements can get a bit clunky when you are checking conditions that run either or.